packages/terra-arrange/src/Arrange.scss

```diff
@@ -7,21 +7,21 @@
   // Vertically aligns arrange content to the top by default
   .arrange {
     display: flex;
+    word-wrap: break-word;
```

Comment:
This feels too opinionated for this component as well. Shouldn't the content elements be responsible for how they wrap their content?

Reply:
The content should handle its own style. The arrange is all about structure.

Reply:
I added this for when there is large text in the end containers, specifically as in one of the examples on the issue where the text is one really really long word.  With the styling of flex: 0 0 auto, the second 0 is say to not shrink on the end caps; therefore if have text that is much longer, it won't wrap when hit the width and would just overflow into the fill element.

I attempted to add it to the content I gave to the arrange component; however, it would only work at this level; hence I added it here as putting it on the content given didn't solve the issue.

If I set it to use flex: 0 1 auto for the start/end containers, this will handle the wrapping as now the content knows it has to shrink; however, this causes issues with other use cases as then the fill content (middle) will squish down the end containers if the middle content is too large.